packages/devextreme/js/ui/scroll_view/ui.scrollable.old.js

```diff
@@ -224,6 +226,7 @@ const Scrollable = DOMComponent.inherit({
 
     _createActions: function() {
         this._strategy && this._strategy.createActions();
+        this._visibilityAction = this._createActionByOption('onVisibilityChanged');
```

Comment:
I don't think we need an action here - let's just make a "function" option.
Don't forget to add an option changed handler